### DISCUSSION TITLE
Any tutorial for the robotic manipulation task trained with RL?

### DISCUSSION THREAD

**Discussion by ge79puv** [StanfordVL/BEHAVIOR-1K#1020] [Ideas]
Hey there,

thanks for your great work! I have tested the ReKep demo and navigation_policy_demo in OmniGibson. I am curious about if there is any turotial or demo for the robotic manipulation tasks, which are trained by RL algorithms, such as SAC and PPO? If you the relevant docs, I appreciate you sharing it in the discussion.

Best regards,
Yunfeng

**Comment by ge79puv** [comment]
For example, I notice that in Behavior-1k paper, you have tested RL-VMC, RL-Prim and RL-Prim.Hist three pipelines. It would be great if you could provide the details of these.

**Comment by hang-yin** [comment]
Hi @ge79puv , I'd say @cgokmen is the perfect person to talk to about this. He has a ton of experience working with RL in OmniGibson.

**Reply by ge79puv** [reply]
Thanks for your answer. Hi @cgokmen, could you help to answer this discussion?

**Comment by cgokmen** [comment]
Hi @ge79puv, we do not have code for the examples provided in the paper unfortunately - OmniGibson has evolved a lot since then and we have been unable to maintain those. We do support parallel-environment RL through the SB3VectorEnvironment class. You can configure it with any task you'd like. We don't currently have a good example of how to train with it in the documentation sadly but we will prioritize this for our next docs update.
